packages/toolkit/.vscode/tasks.json

```diff
@@ -101,12 +145,18 @@
             "group": {
                 "kind": "build",
                 "isDefault": true
+            },
+            "presentation": {
+                "close": true
             }
         },
         {
-            "label": "terminate",
+            "label": "extensionTerminate",
             "command": "echo run terminate",
             "type": "shell",
+            "presentation": {
```

Comment:
Question: what does this do?

Reply:
closes the terminal after it's done instead of leaving it open for other processes.